all/policies/soft_deterministic.py

```diff
@@ -43,7 +43,7 @@ def forward(self, state):
         if self.training:
             action, log_prob = self._sample(normal)
             return action, log_prob
-        return self._squash(normal.loc)
+        return self._squash(normal.loc), torch.as_tensor(0.0, device=self.device)
```

Comment:
Is this the right fix, or would the right fix be to change `SAC`? The log_prob isn't really 0.0, per se. A third option would be to actually compute the log_prob for the greedy action? Still sort of misleading, perhaps.

Reply:
I can totally change that, but my reasoning was this: unless I am misreading the code, the policy is deterministic in eval mode, so the probability of the selected action is really 1, right? So its log would then be 0?